besu/src/main/java/org/hyperledger/besu/cli/config/JwtAlgorithm.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
nit: new files should be Copyright contributors to Hyperledger Besu

Reply:
86895b5